assets/src/stories-editor/components/higher-order/with-keyboard-navigation-handler.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * WordPress dependencies
+ */
+import { withDispatch } from '@wordpress/data';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { UP, DOWN, RIGHT, LEFT, DELETE } from '@wordpress/keycodes';
+/**
+ * Internal dependencies
+ */
+import { ALLOWED_CHILD_BLOCKS, ALLOWED_MOVABLE_BLOCKS } from '../../constants';
+
+const applyWithDispatch = withDispatch( ( dispatch, props, { select } ) => {
+	const {	isReordering } = select( 'amp/story' );
+	const { getSelectedBlock } = select( 'core/block-editor' );
+	const { updateBlockAttributes, removeBlock } = dispatch( 'core/block-editor' );
+
+	const onKeyPress = ( event ) => {
+		const { keyCode, target } = event;
+		const { classList } = target;
+		const selectedBlock = getSelectedBlock();
+
+		if ( ! selectedBlock ) {
+			return;
+		}
+
+		if ( classList.contains( 'editor-rich-text__editable' ) && classList.contains( 'is-selected' ) ) {
+			return;
+		}
+
+		let top = 0;
+		let left = 0;
+		switch ( keyCode ) {
+			case UP:
+				top = -1;
+				break;
+			case DOWN:
+				top = 1;
+				break;
+			case RIGHT:
+				left = 1;
+				break;
+			case LEFT:
+				left = -1;
+				break;
+			case DELETE:
+				removeBlock( selectedBlock.clientId );
```

Comment:
What about the [existing keyboard shortcut](https://github.com/WordPress/gutenberg/blob/4202ab3c7bd150810f9f589eaae39b2a73fb2138/packages/block-editor/src/components/block-editor-keyboard-shortcuts/index.js) for removing blocks?

Reply:
See https://github.com/ampproject/amp-wp/issues/3280